app/controllers/accountants_controller.rb

```diff
@@ -7,19 +7,28 @@ class AccountantsController < ApplicationController
   def index
     patients = pledged_patients.includes(:clinic)
     @patients = paginate_results(patients)
+
+    @entry_name = t 'common.patient'
   end
 
   def search
+    index
+
     @results = if params[:search].present?
-                 Patient.where(pledge_sent: true)
-                        .order_by(pledge_sent_at: :desc)
-                        .search(params[:search])
+                 pledged_patients.search(params[:search])
```

Comment:
ah i see, we intended for the accountants to be able to browse only the last 6 months. edge case: if you search for nothing, this will return everything. do we actually want that to only return the last 6 months, or is it fine to treat that as the initial case?